iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -53,13 +53,13 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
     }
     m_portPool = maybePortPool.value();
 
-    auto maybeIntrospectionMemoryManager = m_roudiMemoryInterface->introspectionMemoryManager();
-    if (!maybeIntrospectionMemoryManager.has_value())
+    auto maybediscoveryMemoryManager = m_roudiMemoryInterface->discoveryMemoryManager();
```

Comment:
Nitpick:

```suggestion
    auto maybeDiscoveryMemoryManager = m_roudiMemoryInterface->discoveryMemoryManager();
```

Reply:
Will change this in the follow up PR

Reply:
Fixed with https://github.com/eclipse-iceoryx/iceoryx/pull/2021/commits/2251d635530ef2b7c44bb1b97aca20fb5662cf9b